packages/design-system-react/CHANGELOG.md

```diff
@@ -7,6 +7,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.0]
```

Comment:
Updating changelog to be a bit more human readable according to Keep a Changelog